BTrees/tests/testConflict.py

```diff
@@ -478,7 +504,7 @@ def testConflictWithOneEmptyBucket(self):
 
     @_skip_wo_ZODB
     def testConflictOfInsertAndDeleteOfFirstBucketItem(self):
-        """
+        """ testConflictOfInsertAndDeleteOfFirstBucketItem
```

Comment:
Hehe, the docstring did mess up the error reporting so I echoed the test name to the top of it and forgot about it. I can turn it into a comment.